py-polars/polars/datatypes.py

```diff
@@ -271,3 +280,92 @@ def pytype_to_polars_type(data_type: Type[Any]) -> Type[DataType]:
     else:
         polars_type = data_type
     return polars_type
+
+
+def polars_type_to_constructor(
+    dtype: Type[DataType],
+) -> Callable[[Optional[str], Sequence[Any]], "PySeries"]:
+    """
+    Get the right PySeries constructor for the given Polars dtype.
+    """
+    map = {
```

Comment:
Can we make these global? Its a bit redundant/expensive to create a hashtable and hash all the values on every Series creation. Especially as this could be called in a hot loop in a `Series.apply(..)` which produces type `List`.

Reply:
Yes, let's do that. I was a bit on the fence; globals is definitely faster, but it's nice for readability to have it together like this. But I'm convinced, let's make the maps global. Will change this.

Reply:
Fixed.